src/app/@ansyn/ol/plugins/annotations/annotations-context-menu/components/annotations-context-menu-buttons/annotations-context-menu-buttons.component.ts

```diff
@@ -158,8 +159,15 @@ export class AnnotationsContextMenuButtonsComponent implements OnInit, AfterView
 	}
 
 	activeChange($event: { label: 'stroke' | 'fill', event: string }) {
-		let opacity = { stroke: 1, fill: 0.4 };
 		const { style } = this.getFeatureProps();
+		let opacity = !!style ? {
```

Comment:
put { stroke: 1, fill: 0.4 } as a variable named 'defaultOpacity' to make it more clear